lib/puppet/reports/servicenow.rb

```diff
@@ -6,12 +6,18 @@
   include Puppet::Util::Servicenow
 
   def process
-    unless status != 'unchanged' || noop_pending
+    decision_parameters = "status: #{status}, "\
+                          "corrective_change: #{corrective_change}, "\
+                          "noop_pending: #{noop_pending}"
+    Puppet.info("servicenow reporting: decision parameters: #{decision_parameters}")
+
+    unless requested_status?(status, corrective_change, noop_pending)
+      Puppet.info('servicenow reporting: decision: no report sent')
```

Comment:
When that message shows up in the log directly below the list of decision_parameters, it's reasonably plain what is meant. When writing to logs there's no need to speak in complete english sentences. You just need to get the point across in a way that's easy to scan.

Reply:
Changed to `Puppet.info(sn_log_entry('decision: Do not create incident'))`
Perhaps if a new function is ever created that returns just the final report type (intentional_change, pending_changes, etc) for consideration, then a new log entry can be placed above this one that prints out that value. Then a troubleshooter will have the params followed by the value, followed by the decision. For now I'v also added an entry that specifies the user selected attributes just in case the logs are being scanned later and it's not easy to find what that setting was at the time the decision was being considered.